### PR TITLE
Fixed default date behavior documentation

### PR DESCRIPTION
**Pull request by Forchapeatl**
- Changes Made : Adjusted  date with time usage.
Fixes [https://github.com/Avaiga/taipy/issues/495](https://github.com/Avaiga/taipy/issues/495)

### CODE REVIEW

**Review by FabienLelaquais — APPROVED** [review]
My first comments were completely wrong.

File: docs/manuals/gui/viselements/date.md_template [hunk 1]
```diff
@@ -34,20 +34,20 @@ a date selector that represents it:
...
-If you don't need to use the date, you can do so:
+If you need to use the date and time, you can do so by setting `with_time` property to `True`:
 
 !!! example "Page content"
 
     === "Markdown"
 
         ```
-        <|{dt}|date|not with_time|>
+        <|{dt}|date|with_time=True|>
```
Comment (FabienLelaquais) [inline docs/manuals/gui/viselements/date.md_template]:
```suggestion
        <|{dt}|date|with_time|>
```

File: docs/manuals/gui/viselements/date.md_template [hunk 2]
```diff
@@ -34,20 +34,20 @@ a date selector that represents it:
...
         ```
   
     === "HTML"
 
         ```html
-        <taipy:date with_time="false">{dt}</taipy:date>
+        <taipy:date with_time="True">{dt}</taipy:date>
```
Comment (FabienLelaquais) [inline docs/manuals/gui/viselements/date.md_template]:
```suggestion
        <taipy:date with_time>{dt}</taipy:date>
```

**Review by FabienLelaquais — APPROVED** [review]
A slight change in the Markdown and HTML element definitions... That's perfect!
I will propagate to the develop branch.
Thanks.

File: docs/manuals/gui/viselements/date.md_template (hunk 1 above)
Comment (Forchapeatl) [reply docs/manuals/gui/viselements/date.md_template]: @FabienLelaquais , thank you for the resourceful suggestion. Does this means `with_time` defaults to `True`  ?
image

Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/date.md_template]: No no. What this says is that in the Markdown syntax used by Taipy, setting a property with no value means True, that's all.

**Review by FabienLelaquais — APPROVED** [review]
👌

File: docs/manuals/gui/viselements/date.md_template (hunk 2 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/date.md_template]: The same remark for the HTML syntax. No value means True.

**Pull request merged**